app/views/organisations/index.html.erb

```diff
@@ -11,13 +11,11 @@
     } %>
 
     <form class="filter-organisations-list__form" data-filter="form" role="search" aria-label="Departments, agencies and public bodies">
-      <%= render "govuk_publishing_components/components/input", {
-        label: {
-          text: "Search for a department, agency or public body"
-        },
-        name: "search-box",
+      <%= render "govuk_publishing_components/components/search", {
```

Comment:
What was the justification for this change? Is it to do a better job of presenting it to assistive tech as a search field? I'm not sure it's a great change as it introduces a search button that wasn't there before, but the searching is interactive/live and the button serves no purpose. It doesn't seem ideal to have an effectively dead button on the page. @injms thoughts?

Reply:
This is _great_ point, it felt like this worked but of course this button is redundant, I know it's not user-facing but the [search in the gem](https://components.publishing.service.gov.uk/component-guide) works a similar way acting a _dynamic-filter-search_ rather than a _click-search_ 

I feel like the Design is correct in the sense the search _icon_ should be there but it should of change to clear icon upon keydown (of course this doesn't exist).

🤔 

Currently exploring other uses of search across govuk / Design system (this is probably worth reverting)

Reply:
Reverted (will rebase before merge to keep clarity for review)
Will raise an enhancement suggestion on the gem and ref on the [Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/132)

Reply:
Similar use on blur within search:

![ezgif-7-4d6d10c0e4d6](https://user-images.githubusercontent.com/71266765/102382361-48a65e00-3fc2-11eb-8848-571b83bd1aa9.gif)